mentors/src/data_pipeline/preprocess_data.py

```diff
@@ -5,16 +5,25 @@
 import pandas as pd
 
 import ibm_transcript_service as transcript_service
-import constants
+import utils
```

Comment:
I'd rather see `constants` remain and just add utils functions to a distinct file `utils.py`